lib/galaxy/tools/parameters/basic.py

```diff
@@ -1648,12 +1648,10 @@ def __init__(self, tool, input_source, context=None):
         if drill_down_dynamic_options is not None:
             self.is_dynamic = True
             self.dynamic_options = drill_down_dynamic_options.code_block
-            self.filtered = drill_down_dynamic_options.filters
```

Comment:
I think both of the above lines were broken since commit c92f6dd47ca , this one should have used `.from_filters()`

Reply:
A good catch but this isn't why the servers are broken I believe - that commit is only in dev.

Reply:
I wasn't implying that, and that tool is abandonware anyway.